### ISSUE TITLE
'hasNextPage' in markdown-remark Not Working as Expected

### ISSUE THREAD

**Issue by madelyneriksen** [gatsbyjs/gatsby#10497]
## Description

I wanted to create pagination using `hasNextPage` in the `allMarkdownRemark` `pageInfo` field. It normally works perfectly, but I found in an instance where the limit is exactly equal to the number of pages after the skip, it does not work (ex. Use four pages, skip two, limit two: hasNextPage should be `false`, but is `true`)

### Steps to reproduce

Here's [the example project](https://github.com/madelyneriksen/Gatsby-hasNextPage-Bug), and here is the query giving me troubles:

```javascript
// There are two markdown files I'm sourcing.

graphql`
query {
  allMarkdownRemark(skip: 1, limit: 1) {
    pageInfo {
      hasNextPage
    }
  }
}`
```

In this example, `hasNextPage` should return a boolean `false`, but instead returns `true`.

### Expected result

`hasNextPage` should return `false` when limit == edges after skipping.

### Actual result

`hasNextPage` returns `true` in this case.

### Environment

```
  System:
    OS: Linux 4.19 Solus 3.9999
    CPU: (8) x64 Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz
    Shell: 5.6.2 - /bin/zsh
  Binaries:
    Node: 10.14.1 - ~/.nvm/versions/node/v10.14.1/bin/node
    npm: 6.4.1 - ~/.nvm/versions/node/v10.14.1/bin/npm
  Browsers:
    Firefox: 64.0
  npmPackages:
    gatsby: ^2.0.33 => 2.0.67 
    gatsby-source-filesystem: ^2.0.12 => 2.0.12 
    gatsby-transformer-remark: ^2.1.15 => 2.1.15 
  npmGlobalPackages:
    gatsby-cli: 2.4.6
```

(Sorry about the formatting, `xclip` seems to have messed up on the underlines)

**Comment by sidharthachatterjee** [comment]
We've fixed this in gatsby@2.0.68, @madelyneriksen 

Thank you so much for reporting this 🙂

**Comment by madelyneriksen** [comment]
Thank you so much! I really appreciate it.
